model-testkit/shared/src/main/scala/explore/boopickle/package.scala

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
Heh. I wondered how adding debug information for the moon phase could require modifying 19 files.